.github/workflows/backport.yml

```diff
@@ -14,6 +14,7 @@ permissions:
 
 jobs:
   backport:
+    if: ${{ contains(github.event.comment.body, '/backport to') }}
```

Comment:
Did you consider `startsWith()`?

Reply:
Yes, it has issues when you have whitespace or other text in front of the backport command.

Reply:
Cool; thanks. I was never sure if it was intentional to allow the command anywhere in the text or not.